x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -401,7 +418,7 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               isInvalid={!destinationIndexNameEmpty && !destinationIndexNameValid}
             />
           </EuiFormRow>
-          {jobType === JOB_TYPES.REGRESSION && (
+          {(jobType === JOB_TYPES.REGRESSION || jobType === JOB_TYPES.CLASSIFICATION) && (
```

Comment:
Is it possible to disable the Create button in the advanced editor if there is a syntax error in the JSON:

![image](https://user-images.githubusercontent.com/7405507/69652580-cb801200-1069-11ea-840f-4de3a45bb4dc.png)

Reply:
I'll add this to the meta issue as it occurs for all types of analytics jobs. I'll do it in a follow-up as a description field also needs to be added to the creation form for all jobs 👌